aten/src/ATen/cpu/FlushDenormal.cpp

```diff
@@ -1,6 +1,11 @@
 #include <ATen/cpu/FlushDenormal.h>
 
+#if defined(CPU_CAPABILITY_AVX512)
```

Comment:
Just including vec.h should work as that file already has this macro check.

Reply:
Thanks, but wouldn't that lead to unnecessary code also being included?

Reply:
I misread this code and code in vec.h. This is fine.